### DISCUSSION TITLE
Need help with Chrome/Edge session location

### DISCUSSION THREAD

**Discussion by mohitume** [sienori/Tab-Session-Manager#1034] [Q&A]
I have dead computer but hd is accessible. I have already checked here C:\Users\USER\AppData\Local\Google\Chrome\User Data\Default\IndexedDB\chrome-extension_apckmmkkalmmljfdlejfiaafhkkhbaeg_0.indexeddb.leveldb but cant find sqlite file as in firefox. There are some .ldb files but they not recognizable in Access or any software as I know. What can be done ?

I have already checked wiki and all discussions. 
https://github.com/sienori/Tab-Session-Manager/wiki/Q&A:-Sessions-data-storage-location;-How-to-migrate-sessions-data-to-another-profile contain only information regarding firefox.

**Comment by mohitume** [comment]
I really appreciate people who write free codes but it is really annoying that they often don't provide basic document/technical document.

**Comment by pedalpoweredsoftware** [comment]
I believe (but can't recall for sure) that the sequence of letters identifying the extension will be different for different installations, but that is possibly the correct folder for the IndexedDB part of the database for this extension. On my computer it is chrome-extension_jkjjclfiflhpjangefhgfjhgfbhajadk_0.indexeddb.leveldb. I do not any files with extensions in that folder, however. The data itself, however, is stored in chrome-extension_jkjjclfiflhpjangefhgfjhgfbhajadk_0.indexeddb.blob (same base name, just with .blob at the end instead of .leveldb.
